fs/react-next-base
# Fix units for IE breakpoints

# Summary

#### A brief description of the pull request.

#### Resolves: #125 

#### Screenshots in case of UI changes

# Test plan

#### List of steps to manually test introduced functionality:

- Go to [ReviewApp](ReviewAppLink)
- Check out SPA window size in IE

#### What do need to pay more attention to, some implicit changes that influence the app:

- ...

# Review notes

#### While reviewing pull-request (especially when it's your pull-request), please make sure that:

- you named PR close to issue name and linked that issue
- you understand what problem is solved by PR and how is it solved
- new tests are in place, no redundant tests
- required ENV variables added and described in `.env.example` and added to Heroku
- associated Heroku review app works correctly with introduced changes
